Casks/y/yuanbao.rb

```diff
@@ -0,0 +1,34 @@
+cask "yuanbao" do
+  version "2.26.10.615,9713c59e07194b783f87213bbf58ec4d"
+  sha256 "3bc2c19414f1c5b754cdfe75392e7b6d68428a06be2b0302bdb302ffb142c18b"
+
+  url "https://cdn-hybrid-prod.hunyuan.tencent.com/Desktop/official/#{version.csv.second}/yuanbao_#{version.csv.first}_universal.dmg"
+  name "Yuanbao"
+  desc "Tencent AI Assistant with Hunyuan and DeepSeek LLMs"
+  homepage "https://yuanbao.tencent.com/"
+
+  livecheck do
+    url "https://yuanbao.tencent.com/api/info/public/general"
+    regex(%r{/cdn-hybrid-prod.hunyuan.tencent.com/Desktop/official/(\h+)/yuanbao_(\d+(?:\.\d+)+)_universal.dmg}i)
+    strategy :json do |json, regex|
+      match = json.dig("pcDownloadUrl", "mac")&.match(regex)
+      next if match.blank?
+
+      "#{match[2]},#{match[1]}"
+    end
+  end
+
+  depends_on macos: ">= :big_sur"
+
+  app "腾讯元宝.app"
+
+  zap trash: [
+    "~/Library/Application Support/com.tencent.yuanbao/",
+    "~/Library/Caches/com.tencent.yuanbao",
+    "~/Library/Containers/com.tencent.yuanbao/",
+    "~/Library/HTTPStorages/com.tencent.yuanbao.binarycookies",
+    "~/Library/HTTPStorages/com.tencent.yuanbao/",
+    "~/Library/Preferences/com.tencent.yuanbao.plist",
+    "~/Library/WebKit/com.tencent.yuanbao/",
```

Comment:
```suggestion
    "~/Library/Application Support/com.tencent.yuanbao",
    "~/Library/Caches/com.tencent.yuanbao",
    "~/Library/Containers/com.tencent.yuanbao",
    "~/Library/HTTPStorages/com.tencent.yuanbao.binarycookies",
    "~/Library/HTTPStorages/com.tencent.yuanbao",
    "~/Library/Preferences/com.tencent.yuanbao.plist",
    "~/Library/WebKit/com.tencent.yuanbao",
```

We usually omit trailing slashes in `zap` entries.